### PR TITLE
Code mirror

### PR DESCRIPTION
**Pull request by BrandonBlack-13**
Iterated through main and commented on CodeMirror 'Webserver.rs' on how the save function works.

### CODE REVIEW

File: server/Cargo.lock
Comment (bjones1) [inline server/Cargo.lock:1]: Don't commit this file -- it should only be committed before a release.

Comment (bjones1) [reply server/Cargo.lock:1]: Either edit your commit history, or just revert changes to this.

**Review by bjones1 — CHANGES_REQUESTED** [review]

**Comment by BrandonBlack-13** [comment]
I added a commit where I just removed the Cargo Lock file. Let me know if this fixed it or if I need to change anything else!

File: client/package-lock.json
Comment (bjones1) [inline client/package-lock.json:1]: Same as cargo.lock -- don't commit this.

**Review by bjones1 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**